src/Target.php

```diff
@@ -286,9 +299,153 @@ public function getEnabled(): bool
      */
     protected function getTime($timestamp): string
     {
-        $toString = \str_replace(',', '.', (string) $timestamp);
+        $toString = \str_replace(',', '.', (string)$timestamp);
         $parts = explode('.', $toString);
 
         return date('Y-m-d H:i:s', $parts[0]) . ($this->microtime && isset($parts[1]) ? ('.' . $parts[1]) : '');
     }
+
+    /**
+     * @param $logVars
+     * @return Target
+     */
+    public function setLogVars(array $logVars): self
+    {
+        $this->logVars = $logVars;
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getLogVars(): array
+    {
+        return $this->logVars;
+    }
+
+    /**
+     * @param bool $microtime
+     * @return Target
+     */
+    public function setMicrotime(bool $microtime): self
```

Comment:
I don't think `bool` is correct type there.

Reply:
The method is poorly named.

The property $microtime controls whether micro seconds are included in the timestamp.
Could you suggest better names from setter/getter ?

Reply:
`enableTimestampMicroseconds()`?

Reply:
`disableTimestampMicroseconds()`?

Reply:
setTimestampMicrosecondsEnabled()?

Reply:
useMicroseconds(bool $value)

Reply:
Look really long ..
withMicroSeconds() // withoutMicroSeconds() ?

Reply:
Another solution would be to allow the user to set the format for timestamps.

Reply:
This another solution sounds much better to me...

Reply:
Implemented.

Default format now includes microseconds, and precision is always 6 numbers after the dot.